test/util.ts

```diff
@@ -11,6 +11,7 @@ import * as _fs from '../lib/util/fs';
 import * as _git from '../lib/util/git';
 import * as _hostRules from '../lib/util/host-rules';
 import { regEx } from '../lib/util/regex';
+import * as _s3 from '../lib/util/s3';
```

Comment:
please don't do that, this will cause all tests to recursively read S3 API. add a new s3.ts file instead.

I've already started to remove the git util import, which improves test time by a few percent.

the problem is that jest is importing this even if it's not used.